### PR TITLE
Add requirements

### PR DESCRIPTION
**Pull request by duncanmmacleod**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,34 +11,56 @@ source:
...
   summary: 'Parameter Estimation automator for gravitational wave analyses.'
   description: |
-    asimov is a python package used to construct automated bots to do various things with parameter estimation jobs for gravitational wave data analysis. There are a number of bits of code which are designed to make writing your own bot easier, and these are importable, but there are also pre-baked bots.
+    asimov is a python package used to construct automated bots to do
+    various things with parameter estimation jobs for gravitational
+    wave data analysis.
+    There are a number of bits of code which are designed to make
+    writing your own bot easier, and these are importable, but there
+    are also pre-baked bots.
   doc_url: https://daniel-williams.docs.ligo.org/supervisor/
```
Comment (transientlunatic) [inline recipe/meta.yaml:64]: Now at https://asimov.docs.ligo.org/asimov/

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,34 +11,56 @@ source:
...
   dev_url: https://github.com/transientlunatic/asimov
```
Comment (transientlunatic) [inline recipe/meta.yaml:65]: Now at https://git.ligo.org/asimov/asimov

**Review by transientlunatic — CHANGES_REQUESTED** [review]
We should update to v0.3.0 while making these changes, and update the documentation and repository locations (I've added these in comments).
Otherwise, thanks for putting this together!

**Comment by duncanmmacleod** [comment]
> We should update to v0.3.0 while making these changes, and update the documentation and repository locations (I've added these in comments).
> Otherwise, thanks for putting this together!

The version here tracks the 'upstream' releases on https://pypi.org/project/ligo-asimov/ which only sees 0.2.0. Do you have plans to upload new release tarballs there, or should we track the gitlab repo?

**Pull request closed without merging**